source/ember-inspector/data.md

```diff
@@ -1,40 +1,33 @@
-If you are using a data library that supports the Ember Inspector such
-as Ember Data, you can inspect your models by clicking on the `Data` menu.
+If you are using Ember Data, you can inspect your models by clicking on the `Data` menu.
 
 When you open the Data tab, you will see a list of model types defined
-in your application. Each model type has the number loaded records next
-to it. You can click on one type to view all of its loaded records.
+in your application, along with the number of loaded records.
+Click on a model type, and the Inspector displays those records.
 
 <img src="../../images/guides/ember-inspector/data-screenshot.png" width="680"/>
 
 ### Inspecting Records
 
-Each row in the list represents a record, with the first four attributes
-as columns (just enough attributes to identify the record). To view all
-of the attributes and more properties, click on the record and it will
-open in the object inspector.
+Each row in the list corresponds to one record. The first four model attributes are shown in the list view. Clicking on the record will open the Object Inspector for that record, and display all attributes.
```

Comment:
I changed this mostly because the original paragraph sounded unwieldy to my ear, but the information is fine. I'm happy to change it back if you prefer.